### PR TITLE
Create it.json

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: src/i18n/it.json [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+    "incorrect_address": "Indirizzo ADM errato",
+    "no_public_key": "Questo indirizzo non è ancora attivo. Impossibile avviare una chat",
+    "me": "Io",
+    "message": "Scrivi un messaggio",
+    "message_mac_os": "Scrivi un messaggio. Cmd + Ctrl + Spazio per il selettore di Emoji",
+    "message_windows_10": "Scrivi un messaggio. Win +. per il selettore Emoji",
+    "my_qr_code": "Il mio codice QR",
+    "new_chat": "Avvia una nuova chat",
+    "no_connection": "Nessuna connessione internet",
+    "no_money": "Not enough tokens. Receive free ADM",
```
Comment (adamant-al) [inline src/i18n/it.json:25]: Translate this string, please

File: src/i18n/it.json [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    "ok": "OK",
+    "partner_info": "Informazioni del partner",
+    "partner_name": "Nome del partner",
+    "received_label": "Recevuto",
+    "recipient": "Indirizzo del partner",
+    "recipient_tooltip": "Indirizzo ADM del partner come U1234567890987654321 o Condividi link",
+    "retry_message": "Impossibile inviare: connessione debole. Fare clic per riprovare",
+    "send_button": "Invia",
+    "send_crypto": "Invia{crypto}",
```
Comment (adamant-al) [inline src/i18n/it.json:34]:
```suggestion
    "send_crypto": "Invia {crypto}",
```

File: src/i18n/it.json [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+    "title": "Chat",
+    "too_long": "Il messaggio è troppo lungo",
+    "free_adm_disclaimer": "ADAMANT è diverso dagli altri messenger: è veramente decentralizzato. Ogni azione, inclusa la messaggistica, ha una tariffa di rete di 0,001 ADM. Per iniziare subito a inviare messaggi, ottieni i tuoi gettoni di benvenuto gratuiti.",
+    "free_adm_title": "Ottieni gettoni ADM gratuiti",
+    "how_to_use_messenger": "Come iniziare con una Blockchain Messenger",
+    "how_to_use_messenger_link": "https://medium.com/adamant-im/how-to-start-with-a-blockchain-messenger-54d1eb9704e6",
+    "transaction_statuses": {
+      "confirmed": "Confirmed in token’s blockchain",
+      "delivered": "Consegnato al nodo",
+      "invalid": "Incorrect information. Check transaction in token's blockchain carefully",
```
Comment (adamant-al) [inline src/i18n/it.json:47]: Translate this string, please

File: src/i18n/it.json [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+      "pending": "Non ancora confermato",
+      "rejected": "La transazione è cancellata o non accettata",
+      "unknown": "Questa criptovaluta non è ancora supportata"
+    },
+    "virtual": {
+      "bitcoin_bet": "Ciao! Sono un bot di scommesse anonimo e dimostrativo di Blockchain 🤑. Accetto scommesse sulla tariffa BTC e pago premi ai vincitori. Scopri di più sul blog di ADAMANT o digita ** / help ** per iniziare a scommettere.",
```
Comment (adamant-al) [inline src/i18n/it.json:53]:
```suggestion
      "bitcoin_bet": "Ciao! Sono un bot di scommesse anonimo e dimostrativo di Blockchain 🤑. Accetto scommesse sulla tariffa BTC e pago premi ai vincitori. Scopri di più sul blog di ADAMANT o digita **/help** per iniziare a scommettere.",
```

File: src/i18n/it.json [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+      "bitcoin_bet_title": "Scommetti sul prezzo di Bitcoin",
+      "bounty_wallet_title": "ADAMANT Bounty Wallet",
+      "bounty_foundation_wallet_title": "Adoption hot Wallet",
+      "exchange_bot": "Ciao! Sono il tuo bot di scambio ₿. Opero istantaneamente e in modo anonimo. Digita ** / help ** per vedere cosa posso fare. ** ℹ ** ️ Ulteriori informazioni sul blog ADAMANT.",
```
Comment (adamant-al) [inline src/i18n/it.json:57]:
```suggestion
      "exchange_bot": "Ciao! Sono il tuo bot di scambio ₿. Opero istantaneamente e in modo anonimo. Digita **/help** per vedere cosa posso fare. **ℹ** ️ Ulteriori informazioni sul blog ADAMANT.",
```

File: src/i18n/it.json [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
... and intentions of using the messenger.\n\nADAMANT is anonymous, your IP is hidden even from partners, and Tor version is available for those that want to take privacy to the next level. But remember, your security and anonymity are up to you. Do not follow links you receive, otherwise your IP can be compromised. Set a password on your device. Learn more about security and anonymity at https://adamant.im/staysecured/.\n\nMake sure you've saved your unique passpharse for this account — logout and login again. Write your passphrase on a sheet of paper as well. Only *you are responsible for the passphrase's safety*. It cannot be recovered. And if other person gets it, your tokens will be stolen and your correspondence will be compromised. Treat this matter with outmost importance, as if tokens in your wallet cost a billion dollars.\n\nTo Start messaging now, *get your free welcoming tokens* on Account tab. Then start a new chat and fill in your friend's ADM address. It is recommended to share your ADM address personally and via ADAMANT only.\n\nSearch Internet for ADAMANT bots' addresses. They can exchange cryptocurrencies, accept bets on crypto rates, and make wishes come true.\n\n",
```
Comment (adamant-al) [inline src/i18n/it.json:61]: Translate this string, please

File: src/i18n/it.json [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+  },
+  "home": {
+    "balance": "Bilanciato",
+    "buy_tokens_anonymously": "Anonimamente sul sito web",
+    "copied": "Copiato",
+    "copy": "Copia",
+    "copy_address": "Copia indirizzo",
+    "copy_uri": "Copia link Condividi",
+    "create_qrcode": "Crea un codice QR",
+    "free_adm_btn": "Free ADM tokens",
```
Comment (adamant-al) [inline src/i18n/it.json:78]: Translate this string, please

File: src/i18n/it.json [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+    "buy_tokens_btn": "Buy & sell tokens",
```
Comment (adamant-al) [inline src/i18n/it.json:79]: Translate this string, please

File: src/i18n/it.json [hunk 9]
```diff
@@ -0,0 +1,303 @@
...
+    "save_button_tooltip": "Salva la tua frase di accesso in un file di testo. Conservalo in un luogo sicuro!",
+    "save_qr_code_to_images": "Salva in immagini",
+    "save_qr_code_tooltip": "Salva la tua frase di accesso come immagine del codice QR. Conservalo in un luogo sicuro!",
+    "scan_qr_code_button_tooltip": "Scansiona il tuo codice QR contenente una frase di access tramite la fotocamera del tuo dispositivo",
+    "subheader": "Messenger decentralizzato"
+  },
+  "login_via_password": {
+    "article": "Blog ADAMANT",
+    "article_hint": "Read about passwords in ",
+    "encoding_data_title": "Crittografia dei dati...",
```
Comment (adamant-al) [inline src/i18n/it.json:110]:
```suggestion
    "encoding_data_title": "Crittografia dei dati…",
```

File: src/i18n/it.json [hunk 10]
```diff
@@ -0,0 +1,303 @@
...
+    "enter_password": "Imposta una password",
+    "incorrect_password": "Password non valida",
+    "login_via_password": "Accedi tramite password",
+    "popup_confirm_text": "Imposta",
+    "popup_hint": "8+ simboli consigliati...",
```
Comment (adamant-al) [inline src/i18n/it.json:115]:
```suggestion
    "popup_hint": "8+ simboli consigliati…",
```

File: src/i18n/it.json [hunk 11]
```diff
@@ -0,0 +1,303 @@
...
+    "popup_title": "Imposta una password",
+    "remove_password": "Remove your password",
```
Comment (adamant-al) [inline src/i18n/it.json:117]: Translate this string, please

File: src/i18n/it.json [hunk 12]
```diff
@@ -0,0 +1,303 @@
...
+  },
+  "nodes": {
+    "active": "Attivo",
+    "fastest_title": "Scegli il nodo più veloce",
+    "fastest_tooltip": "Le richieste verranno elaborate con un nodo più veloce",
+    "host": "Host",
+    "inactive": "Inattivo",
+    "sync": "Sync",
+    "ms": "ms",
+    "nodeLabelDescription": "Supportare la decentralizzazione e migliorare il livello di privacy—<a target=\"_blank\" href=\"https://medium.com/adamant-im/how-to-run-your-adamant-node-on-ubuntu-990e391e8fcc\">run your own ADAMANT node</a>. To add your nodes to the list, you need <a target=\"_blank\" href=\"https://github.com/Adamant-im/adamant-im/\">to deploy web application on separate domain</a>. This limitation refers to Content Security Policy (CSP).",
```
Comment (adamant-al) [inline src/i18n/it.json:132]: Translate this string, please

File: src/i18n/it.json [hunk 13]
```diff
@@ -0,0 +1,303 @@
...
+    "enable_bar": "Titolo della scheda del browser",
+    "enable_bar_tooltip": "Mostra il contatore dei nuovi messaggi nel titolo della scheda",
+    "enable_push": "Notifiche push",
+    "enable_push_tooltip": "Notifiche sul desktop. È necessaria l'autorizzazione",
+    "enable_sound": "Suono",
+    "enable_sound_tooltip": "Riproduci un suono quando viene ricevuto un nuovo messaggio",
+    "stay_logged_in": "Rimani connesso",
+    "stay_logged_in_tooltip": "Imposta una password per accedere più velocemente a questo dispositivo. I tuoi messaggi e la frase d'accesso verranno archiviati crittografati",
+    "format_messages": "Formattazione markdown",
+    "format_messages_tooltip": "Consigliato. Usa _italic_, ** bold **, `monospace` ed altro.",
```
Comment (adamant-al) [inline src/i18n/it.json:161]:
```suggestion
    "format_messages_tooltip": "Consigliato. Usa _italic_, **bold**, `monospace` ed altro.",
```

File: src/i18n/it.json [hunk 14]
```diff
@@ -0,0 +1,303 @@
...
+    "hold_your_device": "Tieni fermo il dispositivo per 2-3 secondi per scansionare il codice QR",
+    "login": {
+      "modal_header": "Scansiona il codice QR contenente una frase d'accesso"
+    },
+    "new-chat": {
+      "modal_header": "Scansiona il codice QR che contiene un indirizzo ADAMANT"
+    },
+    "no_camera_found": "Nessuna fotocamera trovata",
+    "something_wrong": "Qualcosa è andato storto",
+    "waiting_camera": "In attesa della fotocamera ..."
```
Comment (adamant-al) [inline src/i18n/it.json:202]:
```suggestion
    "waiting_camera": "In attesa della fotocamera…"
```

File: src/i18n/it.json [hunk 15]
```diff
@@ -0,0 +1,303 @@
...
+    "status": "Stato",
+    "comment": "Commento",
+    "statuses": {
+      "error": "Errore",
+      "pending": "In sospeso",
+      "registered": "In sospeso",
+      "success": "Success"
+    },
+    "transactions": "Transazioni",
+    "txid": "Transaction #"
```
Comment (adamant-al) [inline src/i18n/it.json:228]:
```suggestion
    "txid": "Transazione #"
```

File: src/i18n/it.json [hunk 16]
```diff
@@ -0,0 +1,303 @@
...
+    "commission_estimate_label": "Commissione stimata di trasferimento",
+    "confirm_approve": "Confirma",
+    "confirm_cancel": "Cancella",
+    "confirm_message": "Conferma il trasferimento di <b>{amount} {crypto}</b> all'indirizzo <span class=\"address-in-confirm\">{address}</span>.",
+    "confirm_message_with_name": "Conferma il trasferimento di <b>{amount} {crypto}</b> a {name} <span class=\"a-text-explanation-enlarged\">({address})</span>.",
+    "confirm_title": "Conferma del trasferimento",
+    "crypto": "Criptovaluta",
+    "decode_from_camera": "Scansiona il QR con la fotocamera",
+    "decode_from_image": "Leggi QR dall'immagine",
+    "error_chat_fee": "Per inviare {crypto} tramite chat,è necessario 0,001 ADM",
```
Comment (adamant-al) [inline src/i18n/it.json:248]:
```suggestion
    "error_chat_fee": "Per inviare {crypto} tramite chat, è necessario 0,001 ADM",
```

File: src/i18n/it.json [hunk 17]
```diff
@@ -0,0 +1,303 @@
...
+  },
+  "votes": {
+    "delegate_description": "Descrizione",
+    "delegate_forged": "Forged",
+    "delegate_link": "Link",
+    "delegate_uptime": "Uptime",
+    "downvote_for_label": "Voto negativo per:",
+    "downvotes": "Voti negativi per",
+    "filter_placeholder": "Nome o indirizzo del portafoglio ADAMANT richiesto",
+    "loading_delegates": "Caricamento delegati...",
```
Comment (adamant-al) [inline src/i18n/it.json:279]:
```suggestion
    "loading_delegates": "Caricamento delegati…",
```

File: src/i18n/it.json [hunk 18]
```diff
@@ -0,0 +1,303 @@
...
+    "table_head_rank": "Grado",
+    "table_head_vote": "Voto",
+    "total_new_votes": "Nuovi voti totali",
+    "total_votes": "Votes total",
+    "upvote_for_label": "Voto per:",
+    "upvotes": "voti positivi",
+    "vote_button_text": "Voto",
+    "vote_request_limit": "Limite di richieste di voto: {limit}",
+    "voting_cost": "Costo del voto",
+    "waiting_confirmations": "La transazione è in corso, sii paziente..."
```
Comment (adamant-al) [inline src/i18n/it.json:301]:
```suggestion
    "waiting_confirmations": "La transazione è in corso, sii paziente…"
```

**Comment by dev-adamant-im** [comment]
No updates

**Pull request closed without merging**